src/libs/actions/IOU.js

```diff
@@ -707,24 +815,16 @@ function deleteMoneyRequest(chatReportID, iouReportID, moneyRequestAction, shoul
     const failureData = [
         {
             onyxMethod: Onyx.METHOD.MERGE,
-            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${chatReportID}`,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${iouReportID}`,
             value: {
-                [optimisticReportAction.reportActionID]: {
+                [optimisticIOUAction.reportActionID]: {
+                    pendingAction: null,
```

Comment:
Do we actually want to clear the `pendingAction` here? Asking because last time we wanted to keep the pendingAction to have the 50% opacity. https://github.com/Expensify/App/blob/293f23b57be2bca3f3eb0af903d5e0c2894cb36a/src/components/OfflineWithFeedback.js#L88.

Old discussion: https://github.com/Expensify/App/pull/17583#discussion_r1174318102